onnxruntime/test/optimizer/qdq_transformer_test.cc

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <cstdint>
```

Comment:
Why is cstdint needed?

Reply:
Added by CPPlint, probably for definition of int64_t/uint64_t or other types. 
It's removed